examples/regexp-find-operations/regexp_find_operations.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+import ballerina/lang.regexp;
+
+function printGroupsWithinLog(regexp:Groups logGroup) {
+    // The first element in the `logGroup` is the entire matched string.
+    // The subsequent elements in `logGroup` represent the captured groups 
+    // (timestamp, component, message).
+    string timestamp = (<regexp:Span>logGroup[1]).substring();
+    string component = (<regexp:Span>logGroup[2]).substring();
+    string logMessage = (<regexp:Span>logGroup[3]).substring();
+
+    io:println(string `Timestamp: ${timestamp}`);
+    io:println(string `Component: ${component}`);
+    io:println(string `Message: ${logMessage}`);
+}
```

Comment:
Let's place all the util functions after the main function, as users might otherwise get distracted.

Reply:
Updated